enterprise/fulltext-addon/src/main/java/org/neo4j/kernel/api/impl/fulltext/lucene/LuceneFulltextFieldEncoding.java

```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU Affero General Public License
```

Comment:
This class can be inlined, the static method has 1 usage which only pass the parameters as is.